style.css

```diff
@@ -0,0 +1,1427 @@
+body {
+  margin: 0;
+  font-family: ARS Maquette, Helvetica Neue, Arial, sans-serif;
+}
+
+.mobile-top-nav {
+  display: none;
+}
+
+.container-768 {
+  display: none;
+}
```

Comment:
## Aim to Stay `DRY`
- [ ] The lines of code on line `#6` & `#10` appear to be similar and make you seem like you are repeating yourself. Would you mind merging them so as to not appear redundant?